IPython/terminal/embed.py

```diff
@@ -19,7 +19,7 @@
 from traitlets import Bool, CBool, Unicode
 from IPython.utils.io import ask_yes_no
 
-class KillEmbeded(Exception):pass
+class KillEmbedded(Exception):pass
```

Comment:
Actually, sorry, I've changed my mind on this one - I think it may be used by third party code, e.g. like this

```python
try:
    for x in range(many):
        f(x)
        embed()
except KillEmbeded:
    pass
```

So I think we should leave the existing name as an alias for a release or two.